### DISCUSSION TITLE
Automatic AWS Instance Creation

### DISCUSSION THREAD

**Discussion by PranathiAT** [learndevopsonline/learndevopsonline#1104] [Q&A]
Hi @r-devops ,

In AWS instances are getting created automatically.Even if i terminate it ,one more instance is getting created.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/129154029/0ff37a4f-340f-4e3a-8f39-bfebd83a069e)

In the above screenshots,unnamed instances are not created by me.Once i login to console i can see one new instance running. Can you please help me on this

**Comment by Ashishch45** [comment]
as per my Knowledge madam which an instance is created with spot instances it can only be stopped not terminate and to terminate we need to go into the spot instances and we need to delete that particular instance which we want to delete as spot instances have terminate option

**Reply by krishnaja-b** [reply]
Hi,
go to spot request, then delete the request. after deleting the request, terminate the instance.
then you don't get instances again.

**Reply by PranathiAT** [reply]
Got it. Thanks
